### PR TITLE
Doc/pgd/update/pgdcliinstallation52

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Rework around pgd cli - work in progress

### CODE REVIEW

File: product_docs/docs/pgd/5/cli/configuring_cli.mdx [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "Configuring PGD CLI"
 navTitle: "Configuring PGD CLI"
 ---
 
-PGD CLI can be installed on any system which is able to connect to the PGD cluster. You will require a user with PGD superuser privileges - the [bdr_superuser role](../security - or equivalent (e.g. edb_admin on BigAnimal PGD) to use PGD CLI.
+PGD CLI can be installed on any system which is able to connect to the PGD cluster. You will require a user with PGD superuser privileges - the [bdr_superuser role](../security - or equivalent (e.g. edb_admin on BigAnimal distributed high-availability) to use PGD CLI.
```
Comment (drothery-edb) [inline product_docs/docs/pgd/5/cli/configuring_cli.mdx:6]:
```suggestion
PGD CLI can be installed on any system which is able to connect to the PGD cluster. You will require a user with PGD superuser privileges - the [bdr_superuser role](../security) - or equivalent (for example, edb_admin on BigAnimal distributed high-availability) to use PGD CLI.
```

File: product_docs/docs/pgd/5/cli/discover_connections.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+PGD CLI can be installed on any system which is able to connect to the PGD cluster. You will require a user with PGD superuser privileges - the [bdr_superuser role](../security) - or equivalent (e.g. edb_admin on BigAnimal distributed high-availability) to use PGD CLI.
+
+## PGD CLI and database connection strings
+
+You may not need a database connection string. For example, when Trusted Postgres Architect installs the PGD CLI on a system, it also configures the connection to the PGD cluster. This means that PGD CLI will automatically connect when run. 
+
+## Getting your database connection string
+
+Every deployment method has a different way of
```
Comment (kelpoole) [inline product_docs/docs/pgd/5/cli/discover_connections.mdx:16]: missing something here

File: product_docs/docs/pgd/5/cli/discover_connections.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+You may not need a database connection string. For example, when Trusted Postgres Architect installs the PGD CLI on a system, it also configures the connection to the PGD cluster. This means that PGD CLI will automatically connect when run. 
+
+## Getting your database connection string
+
+Every deployment method has a different way of 
+
+### For a TPA-deployed PGD cluster
+
+Because TPA is so flexible, you will have to derive your connection string from your cluster configuration file (config.yml). You will need the name or IP address of a host with the role pgd-proxy listed for it. This host will have a proxy you can connect to. Usually the proxy will be listening on port 6432 (check the setting for `default_pgd_proxy_options` and `listen_port` in the config to confirm). The default database name is `bdrdb` (check the setting `bdr_database` in the config to confirm) and the default PGD superuser will be `enterprisedb`.
```
Comment (kelpoole) [inline product_docs/docs/pgd/5/cli/discover_connections.mdx:20]: is the default PGD superuser enterprisedb just for EPAS for for PG and PGE too

Comment (djw-m) [reply product_docs/docs/pgd/5/cli/discover_connections.mdx]: Added content to resolve - postgres, postgres, enterprisedb.....

**Pull request merged**